web/modules/custom/dept_migrate/src/LookupEntry.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Drupal\dept_migrate;
+
+class LookupEntry {
+
+  /**
+   * Lookup data.
+   *
+   * @var false|mixed
+   */
+  protected $item;
+
+  public function __construct(array $lookup_item) {
```

Comment:
Safe to assume this is build for a single lookup but has array set to slot in better with other code? Ie: no implementation to handle multiple items at this time, but could be added another day if really needed?

Reply:
I'm just passing the raw results from the lookup service which always returns an array. I had thought about creating a custom collection type but for the time being we don't really need it.